roles/activemq/molecule/default/molecule.yml

```diff
@@ -4,7 +4,7 @@ dependency:
 driver:
   name: docker
 platforms:
-  - name: activemq
+  - name: activemq-instance
```

Comment:
the mole_it tests are using underscores (role_instance). Can we use that everywhere?

Reply:
forgot underscore as I was mentioning before because this name is used as container/node hostname and it going to break everything

Reply:
true. I forgot about that. I think it's docker driver specific but better avoiding that... I wish I remembered it for my other PR 🙄